shim/src/sanitize.js

```diff
@@ -1,49 +1,7 @@
+import sanitizeAccessors from "./sanitize/accessors";
```

Comment:
I do like this approach, similar to Java and other languages. But I'll go with the flow.